templates/admin/failedJobDetails.tpl

```diff
@@ -14,26 +14,28 @@
 		{translate key=$pageTitle}
 	</h1>
 	<div class="app__contentPanel app__contentPanel__overflow">
-		<pkp-table
-			:columns="columns"
-			:rows="rows"
-			:label="label"
-		>
-			<template #default="{ row, rowIndex }">
-				<table-cell
-					v-for="(column, columnIndex) in columns"
-					:key="column.name"
-					:column="column"
-					:row="row"
-					:tabindex="!rowIndex && !columnIndex ? 0 : -1"
-				>
-					<template #default v-if="column.name === 'value' && isValidJson(row.value)">
-						<span>
-							<pre>{{ row.value }}</pre>
-						</span>
-					</template>
-				</table-cell>
-			</template>
+		<div class="bg-default p-4">
+			<caption v-if="label">
+				<div class="pkpTable__label">
+					<slot name="label">{{ label }}</slot>
```

Comment:
One thing to be super careful about is using classes that are defined in some particular component. These classes should be just used inside that component, not anywhere else. Components should be self-contained as possible. So if we for example refactor some component - it should not impact some UI outside of this component, which is very difficult to notice.

In this case its even class coming from the older Table component, which we intend to remove. So this is good example where this would go wrong.